### DISCUSSION TITLE
Github token in install scripts

### DISCUSSION THREAD

**Discussion by henriquevcosta** [actions/virtual-environments#9973] [General questions]
We are currently building our custom images from Github actions workflows and we're seeing builds occasionally fail with rate limiting (e.g. getting a 403 when getting the latest release of a tool - a public endpoint).
Would it make sense to modify the install scripts in this repo to take a Github token and if provided add it to the REST calls? I can try to find the time to make the PR, I just wasn't sure it was something that made sense for others.

**Comment by mikhailkoliada** [comment]
Hello! We encourage customers to modify the build scripts the way they see fit, though we have no plans to add or support it.

**Reply by henriquevcosta** [reply]
Thanks for the quick reply, first of all!

My question was more if this was something that would be accepted if I submitted a PR for it - I think it could benefit others (and it would make it easier for us to update our scripts in the future) but I'm not sure so I thought I'd ask here.
